packages/bridge-status-controller/package.json

```diff
@@ -78,7 +78,7 @@
   },
   "peerDependencies": {
     "@metamask/accounts-controller": "^28.0.0",
-    "@metamask/bridge-controller": "^21.0.0",
+    "@metamask/bridge-controller": "^22.0.0",
```

Comment:
Wouldn't this require a changelog entry with the "breaking change"?

Reply:
Hmmm yeah not sure why it didn't

Reply:
Oh i think probably because the version of bridge-status-controller did not change; unlike the bridge-controller which has assets-controller as peer dep

Reply:
Will close this 🙏  and release again